pkg/handlers/v1/eni.go

```diff
@@ -69,7 +82,38 @@ func (t eniTransformer) Update(event awsConfigEvent) (Output, bool, error) {
 		return Output{}, false, err
 	}
 
-	return output, filter(config), nil
+	//return output, filter(config), nil
+
+	addedChange := Change{ChangeType: added}
+	deletedChange := Change{ChangeType: deleted}
+	// If an update was detected, check to see if any changes to the NetworkInterfaces occurred
+	for k, v := range event.ConfigurationItemDiff.ChangedProperties {
+		if !strings.HasPrefix(k, "Configuration.PrivateIpAddresses.") {
```

Comment:
My brain is a little fuzzy on this, so just wanted to confirm. Will the public IP information always show up under this private ip change attribute? I remember seeing it in `Configuration.Association` also, so I just want to be sure.

Reply:
From what I've seen in ENI events with public IPs, it appears in both places. My decision with using **PrivateIpAddresses.Association** over **Configuration.Association** is that it fits nicer with the existing private IP extraction code and b/c it's the same logic used for public IP extraction in the ec2.go file. I can cycle through some more events and verify once more to be sure.